### PR TITLE
docs: fix typos in readme

### PR DESCRIPTION
**Pull request by xenowits**
Fix some typos in the README

category: docs

ticket: none

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,9 +9,9 @@
 
 This repo contains the source code for the distributed validator client *Charon*; a HTTP middleware client for Ethereum Staking that enables you to safely run a single validator across a group of independent nodes.
 
-Charon is accompanied by webapp called the [Distributed Validator Launchpad](https://github.com/obolnetwork/dv-launchpad), for distributed validator key creation.
+Charon is accompanied by a webapp called the [Distributed Validator Launchpad](https://github.com/obolnetwork/dv-launchpad) for distributed validator key creation.
```
Comment (xenowits) [inline README.md:12]: Since https://github.com/obolnetwork/dv-launchpad is not open-sourced yet, this hyperlink leads to nowhere

File: README.md [hunk 2]
```diff
@@ -37,8 +37,8 @@ go build -o charon
 
 ## Documentation
 
-The [Obol Docs](https://docs.obol.tech/) website it best place to get started.
-Import sections are the [intro](https://docs.obol.tech/docs/intro),
+The [Obol Docs](https://docs.obol.tech/) website is the best place to get started.
+The important sections are the [intro](https://docs.obol.tech/docs/intro),
```
Comment (dB2510) [inline README.md:41]:
```suggestion
The important sections are [intro](https://docs.obol.tech/docs/intro),
```

File: README.md [hunk 3]
```diff
@@ -49,11 +49,11 @@ For detailed documentation on this repo, see the [docs](docs) folder:
 - [Go Guidelines](goguidelines.md): Guidelines and principals relating to go development
 - [Contributing](contributing.md): How to contribute to charon; githooks, PR templates, etc.
 
-For source code documentation, there is always the [charon godocs](https://pkg.go.dev/github.com/obolnetwork/charon).
+There is always the [charon godocs](https://pkg.go.dev/github.com/obolnetwork/charon) for the source code documentation.
```
Comment (xenowits) [inline README.md:52]: Also, https://pkg.go.dev/github.com/obolnetwork/charon doesn't lead anywhere

File: README.md (hunk 1 above)
Comment (corverroos) [reply README.md:12]: Maybe rather remove that statement in that case

File: README.md (hunk 3 above)
Comment (corverroos) [reply README.md:52]: It works now

**Review by corverroos — APPROVED** [review]

**Review by dB2510 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (xenowits) [reply README.md:12]: @corverroos What about this one?

**Pull request merged**